### DISCUSSION TITLE
https://seaofjapan.page/

### DISCUSSION THREAD

**Discussion by giscus** [sdk-xyz/blog-comments#13] [General]
# https://seaofjapan.page/

https://seaofjapan.page/

**Comment by sdk-kr** [comment]
日本語 - Japanese>

▶ 「東海」とは何ですか？ : 海の名前です。 例）イギリスとヨーロッパの間の「北海」。
▶ 「東海」の位置は？：アジアの大韓民国と日本の間にあります。
▶ "東海"の意味：大陸の東側にあるので東海と呼びます。
(例）大陸中心表記法では、海の西側にある国の地名を海の名前として使います。例えば、西側にロシアのオホーツクがあるため、「オホーツク海」という海の名前が生まれました。

**Comment by sdk-kr** [comment]
< English - OFFICIAL >

▶ What is the "East Sea"? : It is the name of a sea. (e.g., The "North Sea" between the United Kingdom and Europe)
▶ Where is the "East Sea" located? : It is situated between South Korea and Japan in Asia.
▶ Meaning of "East Sea" : It is called the East Sea because it is located to the east of the continent.
(e.g., In continent-centric notation, the names of seas are often derived from the location of countries to the west of the sea. For example, the "Sea of Okhotsk" was named after Okhotsk, Russia, which is located to the west of the sea.)

**Comment by sdk-kr** [comment]
![Screenshot 2023-08-22 at 10 39 30 PM](https://github.com/sdk-xyz/blog-comments/assets/16316626/0cf1efaa-2924-4c92-9655-3c77f85a7fab)
https://www.kado.net/news/articleView.html?idxno=1172844

**Comment by sdk-kr** [comment]
![Screenshot 2023-08-22 at 10 40 51 PM](https://github.com/sdk-xyz/blog-comments/assets/16316626/1cdbbf1a-48d4-4da9-87c8-2d596ea04f2a)
https://www.donga.com/news/article/all/20140313/61666228/1
